### DISCUSSION TITLE
Corrupted characters for polish

### DISCUSSION THREAD

**Discussion by iomidev2** [dompdf/dompdf#3172] [General]
Hey,

i use the custom font for PDFs. The font is loaded, but some polish characters are corrupted. I have checked if the used font supports these characters and it works for simple html files. Any solution for this?

I have the meta in html `<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
`

**Comment by bsweeney** [comment]
In addition to ensuring appropriate encoding, you have to style your content with a font that supports the characters in the text. The bundled DejaVu fonts will work:

```
<p style="font-family: DejaVu Sans;">Ola szykuje się do szkoły. Jest już w piątej klasie. Dawniej obawiała się szkoły, teraz bardzo lubi tam chodzić. W szkole nie tylko uczy się ciekawych rzeczy – spotyka też swoich kolegów i koleżanki. Najbardziej lubi spędzać czas ze swoimi przyjaciółmi z klasy – są to Monika i Michał. </p>
```

The wiki has relevant info:
- [About Fonts](https://github.com/dompdf/dompdf/wiki/About-Fonts-and-Character-Encoding)
- [Unicode How-To](https://github.com/dompdf/dompdf/wiki/UnicodeHowTo)
